CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Gravity is an open source project.
+
+It is the work of tens of contributors. We appricate your help!
+
+We follow a [code of conduct](./CODE_OF_CONDUCT.md).
+
+
+## Filing an Issue
+
+Security issues should be reported directly to security@gravitational.com.
+
+If you are unsure if you've found a bug, consider searching
+[current issues](https://github.com/gravitational/gravity/issues) or
+asking in the [community forum](https://community.gravitational.com/) first.
+
+Once you know you have a issue, make sure to fill out all sections of the
+one of the templates at https://github.com/gravitational/gravity/issues/new/choose.
+
+Gravity contributors will triage the issue shortly.
+
+
+## Contributing A Patch
+
+If you're working on an existing issue, respond to the issue and express
+interest in working on it.  This helps other people know that the issue is
+active, and hopefully prevents duplicated efforts.
+
+If you want to work on a new idea of relatively small scope:
+
+1. Submit an issue describing the proposed change and the implementation.
+2. The repo owners will respond to your issue promptly.
```

Comment:
Same concern as above about making promises without knowing that they'll be upheld.

This line was verbatim from teleport: https://github.com/gravitational/teleport/blob/77e8b63470c14290a7d15caefc990c2c5bc1af52/CONTRIBUTING.md so if we change stuff here, we should adjust it there too.